### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by cjdc007**
* fix lint errors

* run lint on pre-commit hook

### CODE REVIEW

File: src/configs/database.config.js [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
-		return `${host}/test-${dbName}`
-	}
-	return `${host}/${dbName}`;
-}
+const config = (opts = {}) => {
+  const uri = getConnectionUrl(false);
+  mongoose
+    .connect(
+      uri,
+      { ...opts, useNewUrlParser: true, useUnifiedTopology: true },
```
Comment (patrickpingol) [inline src/configs/database.config.js]: Make the unwrapping last so we can override the defined options if we want to.
```suggestion
      { useNewUrlParser: true, useUnifiedTopology: true, ...opts },
```

File: src/controllers/docs.controller.js [hunk 2]
```diff
@@ -1,8 +1,6 @@
-'use strict';
 const path = require('path');
-const packageJson = require('../package.json');
 
-//TODO: Remove initial function
+// TODO: Remove initial function
```
Comment (patrickpingol) [inline src/controllers/docs.controller.js]: 🔥 Unnecessary comment

**Review by llagong — APPROVED** [review]
looks good to me

**Review by patrickpingol — APPROVED** [review]
Please squash and merge. Thanks! 🙇

**Pull request merged**